app/models.py

```diff
@@ -138,10 +138,21 @@ class Supplier(db.Model):
 
     clients = db.Column(JSON, default=list)
 
+    def get_service_counts(self):
```

Comment:
Do we need a query here or can we use existing "fetch all" and just group in code? 

Also is this query properly indexed? how does it perform? Counts in postgres can be slow.